test/unit-tests/components/views/messages/DateSeparator-test.tsx

```diff
@@ -48,6 +48,7 @@ describe("DateSeparator", () => {
             <MatrixClientContext.Provider value={mockClient}>
                 <DateSeparator {...defaultProps} {...props} />
             </MatrixClientContext.Provider>,
+            { legacyRoot: true },
```

Comment:
Why is this option needed in some places?

Reply:
To make this diff smaller, another PR will go and try and fix usage of legacyRoot as to not blow this PR to 3x